subcommand/server-acl-init/command_test.go

```diff
@@ -22,10 +22,22 @@ import (
 
 var ns = "default"
 var releaseName = "release-name"
+var resourcePrefix = "release-name-consul"
```

Comment:
I've updated the tests to use both the `-release-name` and `-resource-prefix` flags since the corresponding change to the helm chart will use both flags and I thought it best to match the Helm chart usage.

I've also updated one test to not use the `-resource-prefix` flag, to test that behaviour (should default to release-name-consul). That's important because if someone upgrades consul-k8s and not the Helm chart, they won't be setting that -resource-prefix flag.